packages/frontend/src/components/Gallery.tsx

```diff
@@ -279,7 +279,14 @@ export default class Gallery extends Component<
       currentTab !== 'files' && currentTab !== 'webxdc_apps'
 
     return (
-      <div className='media-view'>
+      <div
+        // FYI the role="tablist" element is not rendered when
+        // `props.chatId === 'all'`.
+        role={this.props.chatId === 'all' ? undefined : 'tabpanel'}
+        aria-labelledby='tab-media-view'
```

Comment:
that element also does not exist in all media view, does it?

Reply:
Yes. I wanted to keep it in case something changes.

I guess I'll remove this as well.